blueman/services/meta/SerialService.py

```diff
@@ -76,5 +76,14 @@ def connect(self, reply_handler=None, error_handler=None):
                 raise e
         return True
 
-    def disconnect(self, port_id):
-        Mechanism().close_rfcomm('(d)', port_id)
+    def disconnect(self, port_id, reply_handler=None, error_handler=None):
+        try:
+            Mechanism().close_rfcomm('(d)', port_id)
```

Comment:
I guess that's due to the `release_rfcomm_device` call failing in mechanism, no?

Reply:
Not specifically but it would get caught. Any failure in the proxy will be caught here and if there is a handler have it shown in the manager.